arbotix_python/CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog for package arbotix_python
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.11.0 (2020-12-25)
```

Comment:
please revert the changes to the CHANGELOG - this is automatically generated when I do the release.

Reply:
done.